src/RequestHelpers.ts

```diff
@@ -0,0 +1,29 @@
+import {RibbitRequestData, RibbitRequestParams} from './Ribbit';
+import {isPlainObject} from './helpers';
+
+export function buildBody(data: RibbitRequestData): FormData | string {
+    return isPlainObject(data) ? JSON.stringify(data) : (data as FormData);
+}
+
+export function buildUrl(baseUrl: string, params?: RibbitRequestParams): string {
+    if (!params) {
+        return baseUrl;
+    }
+
+    const paramString = Object.entries(params)
+        .map(([key, value]) => {
+            switch (typeof value) {
+                case 'boolean':
+                    return `${key}=${value ? 1 : 0}`;
+                default:
+                    return `${key}=${value}`;
```

Comment:
Likely outside the scope of this ticket, but this could end up being an issue.  If I pass a value containing `&` it will end up creating multiple parameters due to these not being encoded for a URL.

Reply:
Good catch! I have created a ticket for this.